website/addons/wiki/templates/wiki_widget.mako

```diff
@@ -12,9 +12,8 @@
 <% import json %>
 <script>
     window.contextVars = $.extend(true, {}, window.contextVars, {
+        wikiWidget: true,
         usePythonRender: ${json.dumps(use_python_render)},
-        urls: {
```

Comment:
We want to keep the namespacing consistent. Notice that we use `$.extend` to merge this object with the existing context vars. This way, all URLs that are put into contextVars are under the `urls` property.

I went ahead and reverted this during the merge.